packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -328,6 +328,28 @@ export const createNodesForContentType = ({
   }
 
   const createNodePromises = []
+
+  // Create a node for each content type
+  const contentTypeNode = {
+    id: createNodeId(contentTypeItemId),
+    parent: null,
+    children: [],
+    name: contentTypeItem.name,
+    displayField: contentTypeItem.displayField,
+    description: contentTypeItem.description,
+    internal: {
+      type: `${makeTypeName(`ContentType`)}`,
+    },
+    sys: {
+      type: contentTypeItem.sys.type,
+    },
+  }
+
+  // The content of an entry is guaranteed to be updated if and only if the .sys.updatedAt field changed
+  contentTypeNode.internal.contentDigest = contentTypeItem.sys.updatedAt
```

Comment:
```suggestion
    internal: {
      type: `${makeTypeName(`ContentType`)}`,
      contentDigest: contentTypeItem.sys.updatedAt,
    },
    sys: {
      type: contentTypeItem.sys.type,
    },
  }
```

Reply:
Fine. Will improve the structure of this particular part of the code in #30855